### PR TITLE
Add debug logging facilities for fuse

### PR DESCRIPTION
**Pull request by apc999**
With this PR, after setting `log4j.logger.alluxio.fuse.AlluxioFuseFileSystem=DEBUG` in `conf/log4j.properties`, one can find debug logs on FUSE operation in `logs/fuse.log`:

```
2020-03-03 14:33:35,129 DEBUG AlluxioFuseFileSystem - Enter: chmod(path=/aaa,mode=100644)
2020-03-03 14:33:35,131 DEBUG AlluxioFuseFileSystem - Exit (0): chmod(path=/aaa,mode=100644) in 2 ms
2020-03-03 14:33:35,132 DEBUG AlluxioFuseFileSystem - Enter: getattr(path=/aaa)
2020-03-03 14:33:35,135 DEBUG AlluxioFuseFileSystem - Exit (0): getattr(path=/aaa) in 3 ms
2020-03-03 14:33:35,135 DEBUG AlluxioFuseFileSystem - Enter: getattr(path=/._aaa)
2020-03-03 14:33:35,136 DEBUG AlluxioFuseFileSystem - Failed to get info of /._aaa, path does not exist or is invalid
2020-03-03 14:33:35,136 DEBUG AlluxioFuseFileSystem - Exit (-2): getattr(path=/._aaa) in 1 ms
2020-03-03 14:33:35,137 DEBUG AlluxioFuseFileSystem - Enter: getattr(path=/._aaa)
2020-03-03 14:33:35,138 DEBUG AlluxioFuseFileSystem - Failed to get info of /._aaa, path does not exist or is invalid
2020-03-03 14:33:35,138 DEBUG AlluxioFuseFileSystem - Exit (-2): getattr(path=/._aaa) in 1 ms
2020-03-03 14:33:35,138 DEBUG AlluxioFuseFileSystem - Enter: getattr(path=/._aaa)
2020-03-03 14:33:35,140 DEBUG AlluxioFuseFileSystem - Failed to get info of /._aaa, path does not exist or is invalid
2020-03-03 14:33:35,140 DEBUG AlluxioFuseFileSystem - Exit (-2): getattr(path=/._aaa) in 2 ms
2020-03-03 14:33:35,140 DEBUG AlluxioFuseFileSystem - Enter: create(path=/._aaa,mode=100644)
2020-03-03 14:33:35,143 DEBUG AlluxioFuseFileSystem - Exit (0): create(path=/._aaa,mode=100644) in 3 ms
2020-03-03 14:33:35,144 DEBUG AlluxioFuseFileSystem - Enter: getattr(path=/._aaa)
2020-03-03 14:33:35,147 DEBUG AlluxioFuseFileSystem - Exit (0): getattr(path=/._aaa) in 3 ms
2020-03-03 14:33:35,147 DEBUG AlluxioFuseFileSystem - Enter: write(path=/._aaa,buf=jnr.ffi.provider.jffi.DirectMemoryIO[address=0x126223050],size=4096,offset=0)
2020-03-03 14:33:35,166 DEBUG AlluxioFuseFileSystem - Exit (4096): write(path=/._aaa,buf=jnr.ffi.provider.jffi.DirectMemoryIO[address=0x126223050],size=4096,offset=0) in 19 ms
```

### CODE REVIEW

**Comment by LuQQiu** [comment]
@apc999 Please fix the tests, the code changes LGTM

File: integration/fuse/src/main/java/alluxio/fuse/AlluxioFuseUtils.java [hunk 1]
```diff
@@ -209,4 +209,37 @@ private static int getAlluxioErrorCode(AlluxioException e) {
...
+   *
+   * @param logger the logger to use for this call
+   * @param callable the callable to call
+   * @param methodName the name of the method, used for metrics
+   * @param description the format string of the description, used for logging
+   * @param args the arguments for the description
+   * @return the result
+   */
+  public static int call(Logger logger, FuseCallable callable,
+                         String methodName, String description, Object... args) {
```
[the remark this answers is not in the document]
Comment (apc999) [reply integration/fuse/src/main/java/alluxio/fuse/AlluxioFuseUtils.java]: fixed

**Comment by apc999** [comment]
Jenkins, test this please

**Comment by gpang** [comment]
alluxio-bot, test this please

**Comment by apc999** [comment]
alluxio-bot, merge this please

**Pull request merged**